src/esmockModule.js

```diff
@@ -116,7 +117,8 @@ const esmockModuleId = async (parent, treeid, defs, ids, opt, mocks, id) => {
 
   if (!id) return mocks
 
-  const fileURL = opt.resolver(id, parent)
+  const resolver = await esmockResolver(opt)
```

Comment:
the resolver 'getter' calls could be made once inside esmockArgs.js, but either way is Okay really

Reply:
no problem, I'll change it.